tools/shared/src/main/scala/org/scalajs/core/tools/javascript/Printers.scala

```diff
@@ -99,14 +130,20 @@ object Printers {
         // Definitions
 
         case VarDef(ident, rhs) =>
-          print("var ", ident)
-          if (rhs != EmptyTree)
-            print(" = ", rhs)
+          print("var ")
+          print(ident)
+          if (rhs != EmptyTree) {
+            print(" = ")
+            print(rhs)
+          }
 
         case Let(ident, mutable, rhs) =>
-          print(if (mutable) "let " else "const ", ident)
-          if (rhs != EmptyTree)
-            print(" = ", rhs)
+          print(if (mutable) "let " else "const ")
+          print(ident)
+          if (rhs != EmptyTree) {
```

Comment:
Just thought that it would probably be a good idea to either use `rhs eq EmptyTree` or reverse the condition as `EmptyTree != rhs`. As it is now (`rhs != EmptyTree`), this requires a completely useless megamorphic dispatch on `rhs`'s `equals` method.

The same applies to all other comparisons to `EmptyTree` in this file.